lib/internal/main/watch_mode.js

```diff
@@ -33,7 +33,7 @@ markBootstrapComplete();
 // TODO(MoLow): Make kill signal configurable
 const kKillSignal = 'SIGTERM';
 const kShouldFilterModules = getOptionValue('--watch-path').length === 0;
-const kEnvFile = getOptionValue('--env-file');
+const kEnvFile = getOptionValue('--env-file') || getOptionValue('--env-file-if-exists');
```

Comment:
@JonasBa If I'm reading this right, this just makes it so that one file referenced in `--env-file-if-exists` would be watched only if there is no file provided with `--env-file`. But the bug we still have is that you wouldn't be able to specify _both_ an `--env-file` and an `--env-file-if-exists` argument and have _both_ be watched.